config/envs/default.py

```diff
@@ -30,6 +30,11 @@ class DefaultConfig(object):
     AWS_S3_BUCKET_FIND_DOWNLOAD_FILES = os.getenv("AWS_S3_BUCKET_FIND_DOWNLOAD_FILES")
     FIND_SERVICE_BASE_URL = os.getenv("FIND_SERVICE_BASE_URL")
 
+    # Config variables for ACCOUNT STORE
+    ACCOUNT_STORE_API_HOST = os.getenv("ACCOUNT_STORE_API_HOST")
```

Comment:
Can we default this to whatever the hostname is when running it using the docker runner?

Reply:
Defaults to "http://localhost:3003" in `default.py`. Removed `ACCOUNT_STORE_API_HOST` from `development.py`.